boring-sys/build.rs

```diff
@@ -216,10 +291,11 @@ fn main() {
         println!("cargo:rustc-cdylib-link-arg=-Wl,-undefined,dynamic_lookup");
     }
 
-    let include_path = PathBuf::from(
-        std::env::var("BORING_BSSL_INCLUDE_PATH")
-            .unwrap_or_else(|_| String::from("deps/boringssl/src/include")),
-    );
+    let include_path = if cfg!(feature = "fips") {
+        format!("{}/include", BORING_SSL_PATH)
+    } else {
+        format!("{}/src/include", BORING_SSL_PATH)
+    };
```

Comment:
I think I was confused before, `BORING_BSSL_INCLUDE_PATH` was supported even before this PR. The new code should look something like this:
```rust
    println!("cargo:rerun-if-env-changed=BORING_BSSL_INCLUDE_PATH");
    let include_path = std::env::var("BORING_BSSL_INCLUDE_PATH").unwrap_or_else(|| if cfg!(feature = "fips") {
        format!("{}/include", BORING_SSL_PATH)
    } else {
        format!("{}/src/include", BORING_SSL_PATH)
    });
```
Do you want to make that change or should I?

Reply:
Do you mind making the change? I promise to review quickly. :)

Reply:
Pushed in https://github.com/cloudflare/boring/pull/52/commits/0e742156f58dc6ff4846efaefb4f95c17c5549ab.